host/3.0/buster/amd64/node/node10/node10-appservice.Dockerfile

```diff
@@ -1,33 +1,5 @@
-# Build the runtime from source
 ARG HOST_VERSION=3.0.15584
```

Comment:
we should remove hardcoded `HOST_VERSION` in all these files

Reply:
I am still unsure if the ENV var for Host_Version is required anywhere. I did not see anything in Host that requires it (Other than the Proto schema linked below), so I wanted to use this PR to ask if anyone objected to me removing it.

https://github.com/Azure/azure-functions-host/blob/852eed9ceef6ef56b431428a8eb31f1bd9c97f3b/src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto